CHANGELOG.md

```diff
@@ -18,6 +19,7 @@ All notable changes to this project will be documented in this file.
 - Expose PlotElement's TextColor property on WPF.Axes.Axis (#452)
 
 ### Changed
+- Changed OxyPlot.Series.HeatMapSeries.GetValue (#256)
```

Comment:
I think this can be improved. What was changed? I suggest "Fix NaN bug in HeatMapSeries displayed values" or something like that